auto_submit/lib/service/validation_service.dart

```diff
@@ -204,14 +204,14 @@ class ValidationService {
       bool processed = await processMerge(config, result, messagePullRequest);
       if (processed) {
         await pubsub.acknowledge('auto-submit-queue-sub', ackId);
+        log.info('Ack the processed message : $ackId.');
+        github.Issue issue = await githubService.createIssue(
+          github.RepositorySlug('flutter', 'flutter'),
+          'Follow up review for revert pull request $prNumber',
+          'Revert request by author ${result.repository!.pullRequest!.author}',
+        );
+        log.info('Issue #${issue.id} was created to track the review for $prNumber in ${slug.fullName}');
       }
```

Comment:
Do we need an `else` logic?

Reply:
I don't think so. I can add a log info message.

Reply:
Added a log message on failure.